apps/web/src/design-system/icons/index.ts

```diff
@@ -84,6 +85,7 @@ export { Cloud } from './general/Cloud';
 export { Condition } from './general/Condition';
 export { RemoveCondition } from './general/RemoveCondition';
 export { Warning } from './general/Warning';
+export { Logout } from './general/Logout';
```

Comment:
Is this a duplicate export here?

Reply:
Oh yes! thanks for pointing it out, working on it

Reply:
edited the file, can you please check it?